### PR TITLE
cmd/geth, ethdb/pebble: improve database statistic

### PR DESCRIPTION
**Pull request by hadv**

### CODE REVIEW

File: cmd/geth/dbcmd.go [hunk 1]
```diff
@@ -407,7 +407,7 @@ func checkStateContent(ctx *cli.Context) error {
 	return nil
 }
 
-func showLeveldbStats(db ethdb.KeyValueStater) {
```
Comment (rjl493456442) [inline cmd/geth/dbcmd.go:410]: It doesn't make sense to only change the function name, as this function is apparently bound to LevelDB.

Instead, I would prefer to improve this function to also support Pebble.

Comment (hadv) [reply cmd/geth/dbcmd.go:410]: Yes! By the way, I think that it still query and show all db metrics from Pebble already, not only bound to LevelDB!

**Comment by rjl493456442** [comment]
I am considering whether we should return all database statistics instead of just one particular property. From a UX perspective, it is nearly impossible for users to know which database properties are available for dumping. Additionally, LevelDB and Pebble have different statistic definitions, which could add more complexity.

Will discuss with team and turn this pull request into a meaningful improvement.

**Comment by hadv** [comment]
> I am considering whether we should return all database statistics instead of just one particular property. From a UX perspective, it is nearly impossible for users to know which database properties are available for dumping. Additionally, LevelDB and Pebble have different statistic definitions, which could add more complexity.
> 
> Will discuss with team and turn this pull request into a meaningful improvement.

by the way, if geth still support leveldb then it's still valid to re-open this PR https://github.com/ethereum/go-ethereum/pull/25154 ?

**Comment by karalabe** [comment]
We want to drop leveldb, but I'm guessing anyone who runs an arhcive node might have it in leveldb due to thesheer time it takes to resync. So will drop leveldb when we ship pathdb archive. But we don't want to expand on it.

**Comment by rjl493456442** [comment]
**Leveldb statistics**

```
> debug.chaindbProperty()
 Level |   Tables   |    Size(MB)   |    Time(sec)  |    Read(MB)   |   Write(MB)
-------+------------+---------------+---------------+---------------+---------------
   0   |          2 |     521.87546 |       0.00000 |       0.00000 |       0.00000
   1   |         49 |      98.88743 |       0.00000 |       0.00000 |       0.00000
   2   |        495 |     999.71328 |       0.00000 |       0.00000 |       0.00000
   3   |         26 |      52.53258 |       0.00000 |       0.00000 |       0.00000
-------+------------+---------------+---------------+---------------+---------------
 Total |        572 |    1673.00874 |       0.00000 |       0.00000 |       0.00000
-------+------------+---------------+---------------+---------------+---------------

Read(MB):39.01844 Write(MB):37.55427
BlockCache(MB):6.82367 FileCache:4
MemoryCompaction:0 Level0Compaction:0 NonLevel0Compaction:0 SeekCompaction:0
WriteDelayCount:0 WriteDelayDuration:0s Paused:false
Snapshots:0 Iterators:0
```

**Pebble statistics**

```
> debug.chaindbProperty()
      |                             |       |       |   ingested   |     moved    |    written   |       |    amp   |     multilevel
level | tables  size val-bl vtables | score |   in  | tables  size | tables  size | tables  size |  read |   r   w  |    top   in  read
------+-----------------------------+-------+-------+--------------+--------------+--------------+-------+----------+------------------
    0 |     9  208MB     0B       0 |  0.50 |    0B |     0     0B |     0     0B |     0     0B |    0B |   1  0.0 |    0B    0B    0B
    1 |     0     0B     0B       0 |  0.00 |    0B |     0     0B |     0     0B |     0     0B |    0B |   0  0.0 |    0B    0B    0B
    2 |    34   51MB     0B       0 |  0.93 | 1.8KB |     0     0B |     1  191MB |     1  1.9MB | 1.9MB |   1 1052 |    0B    0B    0B
    3 |   271  327MB     0B       0 |  0.72 | 192MB |     0     0B |     0     0B |     3  194MB | 194MB |   1  1.0 |    0B    0B    0B
    4 |   737  1.5GB     0B       0 |  0.45 | 191MB |     0     0B |     0     0B |     1  192MB | 381MB |   1  1.0 |    0B    0B    0B
    5 |   697  102MB     0B       0 |  0.00 |  20MB |     0     0B |     0     0B |   196   21MB |  24MB |   1  1.0 | 8.8MB  20MB  24MB
    6 |  123K  232GB     0B       0 |     - |    0B |     0     0B |     0     0B |     0     0B |    0B |   1  0.0 |    0B    0B    0B
total |  125K  234GB     0B       0 |     - | 5.9KB |     0     0B |     1  191MB |   201  409MB | 601MB |   6 71606 | 8.8MB  20MB  24MB
---------------------------------------------------------------------------------------------------------------------------------------
WAL: 1 files (5.9KB)  in: 5.8KB  written: 5.9KB (0% overhead)
Flushes: 0
Compactions: 11  estimated debt: 839MB  in progress: 0 (0B)
             default: 10  delete: 0  elision: 0  move: 1  read: 0  rewrite: 0  multi-level: 6
MemTables: 1 (256KB)  zombie: 1 (256MB)
Zombie tables: 0 (0B)
Backing tables: 0 (0B)
Virtual tables: 0 (0B)
Block cache: 125K entries (90MB)  hit rate: 0.8%
Table cache: 125K entries (95MB)  hit rate: 70.6%
Secondary cache: 0 entries (0B)  hit rate: 0.0%
Snapshots: 0  earliest seq num: 0
Table iters: 0
Filter utility: 72.3%
Ingestions: 0  as flushable: 0 (0B in 0 tables)

null
```

**Review by rjl493456442 — APPROVED** [review]

**Pull request merged**